src/pre_commit_terraform/terraform_docs_replace.py

```diff
@@ -27,6 +28,14 @@ def main(argv=None):
     parser.add_argument('filenames', nargs='*', help='Filenames to check.')
     args = parser.parse_args(argv)
 
+    warnings.warn(
+        '`terraform_docs_replace` hook is DEPRECATED.'
+        'For migration instructions see '
+        'https://github.com/antonbabenko/pre-commit-terraform/issues/248'
+        '#issuecomment-1290829226',
```

Comment:
There is a linter in my branch, which ask to avoid implicit string concatenations, so, let's not make it less happy (no matter that this hook is not supported anymore)
```suggestion
        'For migration instructions see '
        + 'https://github.com/antonbabenko/pre-commit-terraform/issues/248'
        + '#issuecomment-1290829226',
```

Reply:
You missed one line in the code suggestion. Also, we talked about disabling that rule, so I suggest skipping this change. Besides, this isn't exactly in the scope of the PR as the implicit concatenation was used in `__init__.py` and I didn't change that.

Reply:
Yep, I run test and explitit concatenation is =< 1 * 10^-8 more time consuming than implisit
I don't think that it's a big deal in our case, but i'm good to merge it now as is

Reply:
> is =< 1 * 10^-8 seconds more time consuming
> I don't think that it's a big deal in our case, but i'm good to merge it now as is

Jeez, dude, you're throwing nanoseconds around like a drunken sailor 🤣